### PR TITLE
Test coverages staged ledger diff

### PR DESCRIPTION
**Pull request by polarbeargo**
**Summary of changes**
Changes introduced in this pull request:
- 
Add test coverages for the staged_ledger_diff.rs 
_Originally posted by @jspada in https://github.com/ChainSafe/mina-rs/pull/94#discussion_r764134519_
- [ ] test: Implement test_CoinBaseFeeTransfer().
- [ ] test: Implement test_CoinBaseBalanceData(). 
- [ ] test: Implement test_FeeTransferBalanceData().
- [ ] test: Implement test_CoinBase().
- [ ] test: Implement test_TransactionStatus().
- [ ] test: Implement test_SnappCommand().
**Refere
Screenshot (379)
nce issue to close (if applicable)**

Closes 

**Other information and links**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -4,14 +4,14 @@ members = [
 	"consensus",
 	"crypto",
 	"snark-keys",
+	"base/test-staged_ledger_diff",
```
Comment (hanabi1224) [inline Cargo.toml:7]: why create a new crate instead of making it part of the base crate?

Comment (polarbeargo) [reply Cargo.toml:7]: I learn from your test-serialization under the protocol folder. Cargo throws me an error suggesting that I should add this in the workspace Cargo.toml

Comment (polarbeargo) [reply Cargo.toml:7]: Thank you for pointing this out. I just fix it.

Comment (polarbeargo) [reply Cargo.toml]: Fix in #122

**Comment by polarbeargo** [comment]
Close this PR replace with #113

**Pull request closed without merging**